import sys


def fatal_error(err_message: str):
    print(err_message)
    sys.exit(1)
